January2022/Java/FindTheTownJudge.java

```diff
@@ -19,4 +19,24 @@ public int findJudge(int N, int[][] trust) {
 		}
 		return -1;
 	}
-}
\ No newline at end of file
+}
+
+//C++ CODE
+
+class Solution {
```

Comment:
@heyankit78 , thanks for raising the PR. Can you please move this code to this file https://github.com/Sunchit/Coding-Decoded/blob/master/January2022/C%2B%2B/Find_the_Town_Judge.cpp

Also don't forget to mention your name in the comments for your solution.

Reply:
@Sunchit Sure